src/components/Header/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+import { Navbar, Nav, Container } from 'react-bootstrap';
+
+const Header = () => {
+  const [name, setName] = useState('');
+  const [image, setImage] = useState('');
+  const [navLink, setNavLink] = useState([]);
+
+  useEffect(() => {
+    axios({
+      method: 'get',
+      url: 'http://localhost:3000/organizations/1/public',
+    }).then((res) => {
```

Comment:
PAra evitar validar 3 veces si res existe, lo crearía con un valor default en el parametro -> res = {}

Reply:
**refactor:** single variable for handling header 
![imagen](https://user-images.githubusercontent.com/75862846/159795878-fe210b33-f9f8-4c84-9add-d95b5c5064d5.png)